### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by juanda99**
Running the example is not clear enough. Also you have to look through code in order to enter the credentials

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,13 +140,19 @@ Admin-on-rest is designed as a library of loosely coupled React components built
 
 Pull requests are welcome. Try to follow the coding style of the existing files, and include unit tests and documentation. Be prepared for a thorough code review, and be patient for the merge - this is an open-source initiative.
 
+## Run the example
```
Comment (fzaninotto) [inline README.md]: No, please remove this title: it breaks the higher section "contributing"

File: README.md [hunk 2]
```diff
@@ -140,13 +140,19 @@ Admin-on-rest is designed as a library of loosely coupled React components built
...
 
 ```sh
+npm install
 make run
 ```
 
 And then browse to [http://localhost:8080/](http://localhost:8080/).
+The credentials are **login/password**
+
+
```
Comment (fzaninotto) [inline README.md]: No need to add more than one line break

File: README.md (hunk 1 above)
Comment (juanda99) [inline README.md]: Maybe move Run the example  before contributing section?

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**